packages/fast-components-react-msft/src/subheading/index.ts

```diff
@@ -0,0 +1,14 @@
+import * as React from "react";
+import { IFoundationProps } from "@microsoft/fast-components-react-base";
+

```

Comment:
why is there a new line between these imports versus the rest?

Reply:
fixed